subprojects/docs/src/samples/testing/junitplatform/engine/build.gradle

```diff
@@ -13,9 +13,11 @@ dependencies {
 }
 // END SNIPPET vintage-dependencies
 
+// START SNIPPET filter-engine
 test {
     useJUnitPlatform {
         // includeEngines 'included-engine'
```

Comment:
It's somewhat confusing that the example uses commented out includes/excludes.

How about having a real engine filtering sample where we use at least one of them without a comment? 

Also unrelated to this commit, but the snippet above is misleading - If you just want vintage, you don't need the jupiter api and engine on the classpath. Seems like we are trying to put too many things into one example.

Reply:
Yes. Previously the mixed JUnit3/4/5 sample and engine sample are put together. Now I've separated them as different samples - 'mix' and 'engine'.

Reply:
Yes. Previously the mixed JUnit3/4/5 sample and engine sample are put together. Now I've separated them as different samples - 'mix' and 'engine'.